demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -1,25 +1,34 @@
 import glob
 import json
+import logging
```

Comment:
this is creating a mix between using `logging` and `print_*` functions in the **same** module.
I believe we should stick to one - `logging` is preferred, need to check the uses of the current functionality in other places (how will it affect it).

Reply:
Fixed the logs to be under `print_*` - as this is the used method over the full module (the logging was a result of the migrated code from content).